.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,54 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ master,  releasebranch_7_8]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    # Check every Saturday at 18:36
+    - cron: '36 18 * * 6'
+
+jobs:
+  analyze:
+    name: ${{ matrix.language }} - Analyze
+    runs-on: ubuntu-20.04
+
+    strategy:
+      fail-fast: false
+      matrix:
+        # C is included in cpp, no separate C language available on CodeQL
+        language: [ 'cpp', 'python' ]
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Initialize CodeQL
+      uses: github/codeql-action/init@v1
+      with:
+        languages: ${{ matrix.language }}
+
+    - name: Get dependencies
+      run: |
+        sudo apt-get update -y
+        sudo apt-get install -y wget git gawk findutils
+        xargs -a <(awk '! /^ *(#|$)/' ".github/workflows/apt.txt") -r -- \
+            sudo apt-get install -y --no-install-recommends --no-install-suggests
+    - name: Create installation directory
```

Comment:
Use either an empty line between steps or no empty line. Not both styles. Internal consistency is probably more important than convenience of some theoretical Meld-like operation in the future.

Reply:
The mix was done intentionally because this way the GRASS GIS specific part appears as one block, separating it from the CodeQL specific steps. I agree on Meld-like convenience, so I will update it.